docs/integrations/destinations/local-csv2.md

```diff
@@ -0,0 +1,34 @@
+# Local CSV
```

Comment:
will replace local-csv.md with this one once we switch to this integration. it would be nice, if we could put this doc in the `airbyte-integrations/csv-destination`.